public/views/projects-source.html

```diff
@@ -41,7 +41,12 @@ <h4 class="site-header" id="{{project.site_id | fixMyWspId}}">{{project.site_nam
               <label for="check{{project.site_id}}{{project.tool_id}}" style="text-indent:20px;font-weight:normal">{{project.tool_name}}</label>
             </div>
           </div>
-          <div class="row noFocus" ng-show="project.selected" id="toolSelDest{{project.tool_id}}" tabindex="-1">
+          <div class="row noFocus" ng-show="!project.hasContentItem" id="hasNoContentItem" tabindex="-1">
```

Comment:
If the  id="hasNoContentItem" is not being used we could omit it. If there is more than one site with no resource content  we will have an id clash. If we are using it later on we could do id="toolSelDest{{project.tool_id}}" to avoid this.